python/nav/web/templates/maintenance/calendar.html

```diff
@@ -11,8 +11,8 @@
       </a>
 
       <form action="{% url 'maintenance-calendar' %}" method="get" class="inline-form">
-        <input type="text" name="year" value="{{ this_month.year }}" size="4" maxlength="4"/>
-        <input type="text" name="month" value="{{ this_month.month }}" size="2" maxlength="2"/>
+          <input type="number" name="month" value="{{ this_month.month }}" min="1" max="12" required id="id_month"/>
+          <input type="number" name="year" value="{{ this_month.year }}" min="2000" max="2100" required id="id_year"/>
```

Comment:
Nitpick: You've reversed the order of the form fields here - please keep the original order :)